src/renderer/components/titlebar/titlebar-profile-dropdown/TitleBarProfileDropdown.tsx

```diff
@@ -15,6 +15,8 @@ import { BiChevronDown } from 'react-icons/bi';
 
 import styles from './TitleBarProfileDropdown.module.css';
 
+import playerData from '../../../../common/player.json';
+
```

Comment:
This makes no sense. JSON files are immutable, that is, you cannot alter it because in this case it is part of the source code itself. For this you need to get a json file inside .openmc/settings.json